src/assets/content/dictionary/user-research.md

```diff
@@ -1,6 +1,10 @@
 ---
 title: User Research
-definition:
-sources: []
-perspectives: []
+definition: User Research is undertaking activities to better understand your target audience, especially in terms of what they need and any pain points to be able to solve those problems - for example generating insights through a survey, interviews, testing et al to inform design solutions.
+sources: 
+- sourceurl: https://www.interaction-design.org/literature/topics/user-research
+perspectives: 
+- meaning: is vital to ensure what we are building takes a human-centred approach empathising with the people we are seeking to serve alongside thinking critically about any decisions being made in terms of whether we have the data to back them up.
```

Comment:
Hi @NyashaDuri! Thank you for contributing to our tech dictionary! This looks pretty good, there is just a little awkward wording here in the perspective with the way the values get formatted: 
<img width="1770" alt="image" src="https://github.com/cherryontech/website/assets/3057984/82b02b5f-a63a-4551-8a18-5f73d30521fb">

Here's a screenshot of the deploy preview where I tested your work. You can view the deploy preview in the PR checks in GitHub to test your own changes. You can also refer to the instructions here for how the values will be formatted: https://github.com/cherryontech/website/blob/pit/CONTRIBUTING.md#tech-dictionary-example

Request a re-review when you've made changes and feel free to ask any questions!